File: program/program/print-pattern-1/print_pattern_1.r
printPattern <- function(n) {
  for (i in 1:n) {
    for (j in 1:i) {
      cat(j, " ")
    }
    cat("\n") # Move to the next line after each row
  }
}

# Test the function with input 5
printPattern(5)

File: program/program/print-pattern-1/printPattern1.scala
object NumberPattern {
  def main(args: Array[String]): Unit = {
    // Taking input from the user
    println("Enter the number of rows:")
    val numRows = scala.io.StdIn.readInt()

    // Printing the pattern
    for (row <- 1 to numRows) {
      // For each row, print numbers from 1 to row
      for (num <- 1 to row) {
        print(s"$num ")
      }
      println() // Move to the next line after printing each row
    }
  }
}
